src/css/vendor/sortable-tables.css

```diff
@@ -0,0 +1,7 @@
+table.sortable th:not(.sorttable_sorted):not(.sorttable_sorted_reverse):not(.sorttable_nosort)::after {
+  content: " \25B4";
+}
+
+table.sortable:hover {
```

Comment:
Can we target the `thead` element? That way the pointer doesn't appear on the whole table